<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>805</width>
    <height>694</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QMdiArea" name="mdiArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>781</width>
      <height>501</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Файл</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>351</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Словарь</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>805</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action_2"/>
    <addaction name="action_4"/>
    <addaction name="action_9"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Работа</string>
    </property>
    <widget class="QMenu" name="menu_4">
     <property name="title">
      <string>Перевод</string>
     </property>
     <addaction name="separator"/>
    </widget>
    <addaction name="menu_4"/>
    <addaction name="separator"/>
    <addaction name="action_10"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="action_2"/>
   <addaction name="separator"/>
   <addaction name="action_4"/>
   <addaction name="action_9"/>
   <addaction name="action_10"/>
  </widget>
  <action name="action">
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_2">
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/folderblue_92960.ico</normaloff>../../Desktop/folderblue_92960.ico</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
  </action>
  <action name="action_4">
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/22.ico</normaloff>../../Desktop/22.ico</iconset>
   </property>
   <property name="text">
    <string>Сохранить файл</string>
   </property>
  </action>
  <action name="action_dict">
   <property name="text">
    <string>Пример.dict</string>
   </property>
  </action>
  <action name="action_9">
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/note_clipboard_notes_icon_188575.ico</normaloff>../../Desktop/note_clipboard_notes_icon_188575.ico</iconset>
   </property>
   <property name="text">
    <string>Создать файл</string>
   </property>
  </action>
  <action name="action_10">
   <property name="icon">
    <iconset>
     <normaloff>../../Desktop/pen_pencil_ruler_icon_188571.ico</normaloff>../../Desktop/pen_pencil_ruler_icon_188571.ico</iconset>
   </property>
   <property name="text">
    <string>Перевести текст</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
